samples/java-eventing-shopping-cart/src/main/java/shopping/cart/view/ShoppingCartViewServiceView.java

```diff
@@ -1,41 +1,27 @@
-/*
- * Copyright 2021 Lightbend Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
+/* This code is managed by Akka Serverless tooling.
+ * It will be re-generated to reflect any changes to your protobuf definitions.
+ * DO NOT EDIT
  */
-
-package shopping.cart;
+package shopping.cart.view;
 
 import com.akkaserverless.javasdk.view.ViewContext;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import shopping.cart.domain.ShoppingCartDomain;
-import shopping.cart.view.ShoppingCartViewModel;
 
-// user view impl
-public class ShoppingCartView extends AbstractShoppingCartView {
+public class ShoppingCartViewServiceView extends AbstractShoppingCartViewServiceView {
 
-  private static Logger LOG = LoggerFactory.getLogger(ShoppingCartView.class);
+  private static Logger LOG = LoggerFactory.getLogger(ShoppingCartViewServiceView.class);
```

Comment:
Ouch... I guess our convention should be not to suffix `Service` to protobuf services? Anyway not directly relevant to this PR.

Reply:
Yeah, I agree it's not great, some related discussions in #238